src/ModelContextProtocol/Server/McpServerPromptTypeAttribute.cs

```diff
@@ -6,9 +6,8 @@ namespace ModelContextProtocol.Server;
 /// Used to attribute a type containing methods that should be exposed as MCP prompts.
 /// </summary>
 /// <remarks>
-/// This is primarily relevant to methods that scan types in an assembly looking for methods
-/// to expose, such as <see cref="McpServerBuilderExtensions.WithPromptsFromAssembly"/>. It is not
-/// necessary to attribute types explicitly provided to a method like <see cref="McpServerBuilderExtensions.WithPrompts{TPrompt}"/>.
+/// It is not necessary to attribute types explicitly provided to a method like <see
+/// cref="McpServerBuilderExtensions.WithPrompts{TPrompt}"/>.
```

Comment:
If we delete WithToolsFromAssembly, we don't need the corresponding McpServerToolsTypeAttribute at all; its sole purpose is to help that method discover what types contain tools. But per my earlier comment, we should think really hard about removing these. They're a really nice convenience / getting started story, and they're perfectly reasonable for folks (likely the majority) that don't need to be NativeAOT compatible.